client/containers/App/app.scss

```diff
@@ -17,6 +17,10 @@
         line-height: 30px;
         background: #c23030;
         z-index: 99999;
+        &:hover {
+            opacity: 0;
+            pointer-events: none;
+        }
```

Comment:
This is such a good idea, however it doesn't seem to work consistently for me on Firefox. Hovering the block triggers `pointer-events: none` which causes the `:hover` css to unattach. I get inconsistent behavior that I think may be dependent on how the browser decides to render toggling between `:hover` and not.

Reply:
Yeah, I noticed that too. I'll either fix or remove it.